tool/js/cells.js

```diff
@@ -0,0 +1,33 @@
+/*****************************************************************
```

Comment:
Did you mean to include this script? Seems like only apps.js and comps.js are used.

Reply:
+1 for this and the others.  Based on their content these shouldn't be on the kappnav side.

Reply:
Good catch. No, that does not belong.  I removed it.